packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.types.ts

```diff
@@ -40,6 +40,10 @@ export interface IScrollablePaneProps extends React.HTMLAttributes<HTMLElement |
   initialScrollPosition?: number;
 
   scrollbarVisibility?: ScrollbarVisibility;
+  /**
```

Comment:
Why is this for the overall scrollable pane? It seems like it would make sense to just set this on the given `<Sticky />` that needs it.